docs/policy/comanage-instructions-user.md

```diff
@@ -0,0 +1,54 @@
+Registering for the OSG COManage
+================================
+
+OSG is using a new identity management system called COManage.
+This system will be used for managing access to OSG staff-internal monitoring webpages at UNL
+and managing access for OASIS logins.
+
+Follow the instructions below to register with COManage and join the OSG group.
+
+!!! note
+    This page is for users who want to register with COManage.
+    If you are a COManage Admin and want to approve a user registration,
+    go to the [Approving COManage Registrations](/policy/comanage-instructions-admin) page instead.
+
+
+1.  Go to <https://registry.cilogon.org/registry/co_petitions/start/coef:64>
+
+1.  Search for your institution and select it.
+
+1.  Log in with your institutional credentials.
+    You should reach the "OSG Registration: Self Signup With Approval" page.
+
+1.  Click "BEGIN".
+
+1.  Enter your name and email address in the form fields;
+    defaults might be filled in from what your institution provided,
+    but you are permitted to change them.
+
+1.  Click "SUBMIT".
+
+1.  Check for an email from <registry@cilogon.org> asking you to verify your OSG registration.
+    Follow the link in the email.
+    You should see an "Invitation to OSG" page.
+
+1.  Click "Accept".
+    You should see a message on top saying "Petition Confirmed".
+
+1.  Wait for approval from a COManage Admin.
+    You will get an email from <registry@cilogon.org> saying "Petition for <NAME> changed status
+    from Pending Approval to Approved".
+    Ignore the links in the email.
+
+1.  To verify your access, go to
+    <https://registry.cilogon.org/registry/users/login>.
+
+    If you see a page saying "Welcome to OSG. Please select an action from the menus.",
+    then your registration was successful.
+
+    Otherwise, if you see the "Self Signup With Approval" page, then your registration was unsuccessful;
+    contact <help@opensciencegrid.org> for assistance.
+
+1.  The menus are on the left.
```

Comment:
Note that this step is optional or just to explore the interface.

Reply:
Should I just drop that step?

Reply:
Fine by me.